web/src/locales/es/data.json

```diff
@@ -380,6 +381,7 @@
     "Is profile public - Tooltip": "Después de estar cerrado, solo los administradores globales o usuarios de la misma organización pueden acceder a la página de perfil del usuario",
     "Modify rule": "Modificar regla",
     "New Organization": "Nueva organización",
+    "Properties - Tooltip": "Propiedades de la organización",
```

Comment:
Reuse `Properties - Tooltip`, you should not add any new words

Reply:
done